pages/Graph/Overview.md

```diff
@@ -16,13 +20,81 @@ This section should also cover how our graph representation could be applied to
 | 0.1     | Draft |
 
 ## Purpose
-
-TBD
+1. Describe the form of Graph Message posted to the blockchain
+1. Explain how to retrieve and resolve the graph
```

Comment:
Should these be different numbers?

Reply:
great question. Markdown will generate consecutive numbers if you use this format.